Update examples and add contributing guide (#283)
bpatel2107/pyrokinetics: docs/howtos/convert.rst

    $ pyro convert CGYRO "my_gs2_file.in" -o "input.cgyro"

Pyrokinetics will automatically detect the type of the input file. Note
that when converting between codes only the data stored in `LocalGeometry`,
`LocalSpecies` and `Numerics` is transferred over. Any other CGYRO input

> parameters are taking from a template file which is taking from `pyrokinetics.template_dir`

Reviewer: ```suggestion
parameters are taking from a template file in `pyrokinetics.template_dir`
```

Worth mentioning how to change this template?

Author: I do mention that later on, is that what you meant?

Reviewer: Oh yes, ignore me, sorry!
